descriptors/ModuleDescriptor-template.json

```diff
@@ -310,6 +310,16 @@
           ],
           "permissionsRequired": []
         },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/marccat/auth-fixed-fields-code-groups",
+          "modulePermissions": [
+            "configuration.all"
+          ],
+          "permissionsRequired": []
```

Comment:
no required permissions? This means that the API can be called w/o logging in... Is that really a requirement?

Reply:
I have changed the permissions. Sorry.